src/main/java/ca/on/oicr/gsi/dimsum/FrontEndConfig.java

```diff
@@ -28,6 +30,11 @@ public class FrontEndConfig {
   private final List<String> pendingStates =
       Stream.of(PendingState.values()).map(PendingState::getLabel).toList();
 
+  private final List<String> completedGates =
+      Stream.of(CompletedGate.values()).map(CompletedGate::getLabel).toList();
+
+  private final List<String> stopStatus = Arrays.asList("Yes", "No");
```

Comment:
These don't need to be in the config - the values can be hard-coded in the typescript instead of hard-coding them here